ui/imports/shared/views/chat/MessageView.qml

```diff
@@ -552,7 +552,8 @@ Loader {
                 }
 
                 onReplyMessageClicked: {
-                    root.messageStore.messageModule.jumpToMessage(root.responseToMessageWithId)
+                    if (!root.quotedMessageDeleted && !(!root.quotedMessageText && contentType !== StatusMessage.ContentType.Image))
```

Comment:
I see that you copied line 670, but I think I spotted an issue with the original logic.

First, `contentType` is for the message itself, not the quotedMessage, so the condition doesn't work if the quotedMessage is an image.

Maybe we can just check `quotedMessageFrom` instead? if there is no `quotedMessageFrom`, then it's already sure that we don't have the quotedMessage.

Another point is can we just disable the MouseArea when we don't have the message?

Reply:
we have a problem in how we process the click, not in the click. So I don't think it's a good idea to disable mouse area.
But yes, logically, `quotedMessageFrom` should works. I will fix original issue as well.

Reply:
fixed. I leave `quotedMessageDeleted` cos before reloading we still have a `quotedMessageFrom`.